packages/notificator/notificator_test.go

```diff
@@ -0,0 +1,126 @@
+package notificator
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+)
+
+type ParseRecipientNotificationsItem struct {
+	Input []map[string]string
+	Want  map[int64]*[]notificationRecord
+}
+
+func TestParseRecipientNotifications(t *testing.T) {
+	table := []ParseRecipientNotificationsItem{
+		ParseRecipientNotificationsItem{
+			Input: []map[string]string{
+				map[string]string{
+					"recipient_id": "1",
+					"role_id":      "1",
+					"cnt":          "2",
+				},
+				map[string]string{
+					"recipient_id": "1",
+					"role_id":      "2",
+					"cnt":          "1", //1
+				},
+				map[string]string{
+					"recipient_id": "2",
+					"role_id":      "3",
+					"cnt":          "4",
+				},
+				map[string]string{
+					"recipient_id": "2",
+					"role_id":      "4",
+					"cnt":          "3",
+				},
+			},
+			Want: map[int64]*[]notificationRecord{
+				1: &[]notificationRecord{
+					notificationRecord{
+						RoleID:       1,
+						RecordsCount: 2,
+					},
+					notificationRecord{
+						RoleID:       2,
+						RecordsCount: 1,
+					},
+				},
+				2: &[]notificationRecord{
+					notificationRecord{
+						RoleID:       3,
+						RecordsCount: 4,
+					},
+					notificationRecord{
+						RoleID:       4,
+						RecordsCount: 3,
+					},
+				},
+			},
+		},
+		//========= new item =========
+	}
+
+	for i, item := range table {
+		result, err := parseRecipientNotification(item.Input)
+		if err != nil {
+			t.Error(err, "on ", i, " item")
```

Comment:
we are agreed that we will start testify for all new tests, so assert.NotError(err, "on" + i + "item")